### PR TITLE
P branch

### PR DESCRIPTION
**Pull request by FrancoSpeziali**
I want to merge these changes

### CODE REVIEW

**Comment by neptuneboy666** [comment]
ok

**Comment by SureLife** [comment]
Fantastic work here, Great job!

**Comment by Julz1997** [comment]
Test comment

**Comment by FrancoSpeziali** [comment]
Good use of boilerplate code!

**Comment by Deobap73** [comment]
itś ok

**Comment by BarbaraPapa** [comment]
sounds good

**Comment by SamuelDCI** [comment]
interesting

**Review by SureLife — APPROVED** [review]
approved

**Comment by ThulasiSushma** [comment]
:)

**Review by Gordon-Zoeller — APPROVED** [review]

**Review by Persis07 — APPROVED** [review]
Pretty Interesting

**Review by salmaimam — APPROVED** [review]
Hey, its good

**Review by Deobap73 — APPROVED** [review]
Correct

**Review by SamuelDCI — APPROVED** [review]
interesting

File: index.html [hunk 1]
```diff
@@ -7,6 +7,8 @@
     <title>My Website</title>
   </head>
   <body>
+    <p>Welcome to miss Davis Website</p>
+    <p>Hello, p-branch!</p>
```
Comment (FrancoSpeziali) [inline index.html:11]: There's a typo here!

Comment (FrancoSpeziali) [reply index.html:11]: oops I was wrong, there was no typo!

**Pull request merged**